### ISSUE TITLE
no such command: 'popup'

### ISSUE THREAD

**Issue by vbauerster** [alexherbo2/explore.kak#7]
After updating `connect` plugin, there is following error with this one:
```
Error: 1:1: 'e' backup/: is a directory
error running hook RuntimeError(1:1: 'e' backup/: is a directory)/explore-files: 6:7: 'explore-files' 2:5: '+' 2:5: 'connect' no such command: 'popup'
error running hook RuntimeError(1:1: 'e' backup/: is a directory)/explore-files: 6:7: 'explore-files' 2:5: '+' 2:5: 'connect' no such command: 'popup'
```

**Comment by alexherbo2** [comment]
I recently added the use of `popup` in [connect.kak] for [fzf] commands and documented in the [FAQ – I don’t have a popup command].

[connect.kak]: https://github.com/alexherbo2/connect.kak
[fzf]: https://github.com/junegunn/fzf
[FAQ – I don’t have a popup command]: https://github.com/alexherbo2/connect.kak/blob/master/docs/faq.md#i-dont-have-a-popup-command

TL;DR: Just create an alias if you don’t have a `popup` command.

``` kak
alias global popup your-terminal-command-here
```

**Comment by vbauerster** [comment]
My bad didn't read the faq indeed.
Thank you for quick reply!
